.devcontainer/launch.json

```diff
@@ -0,0 +1,22 @@
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Debug R (C debugging)",
+      "type": "cppdbg",
+      "request": "launch",
+      "program": "/workspaces/r-dev-env/scripts/launch_r.sh",
+      "args": [],
+      "stopAtEntry": false,
+      "cwd": "${workspaceFolder}",
+      "environment": [
+        {
+          "name": "R_RPATH_LINUX",
+          "value": "${config:r.rpath.linux}"
+        }
+      ],
```

Comment:
Should not need this here either. The path to R is only needed when we launch the terminal. The debugger identifies the running R process with the process ID (as returned by Sys.getpid() in R).